msm/src/serialization/witness.rs

```diff
@@ -156,46 +160,119 @@ pub fn deserialize_field_element<Fp: Field, Env: InterpreterEnv<Fp>>(
     }
 
     // FIXME: range check
-    let c0 = Env::get_column_for_msm_limb(0);
-    let c1 = Env::get_column_for_msm_limb(1);
-    let c2 = Env::get_column_for_msm_limb(2);
-    let c3 = Env::get_column_for_msm_limb(3);
-    let c4 = Env::get_column_for_msm_limb(4);
-    let c5 = Env::get_column_for_msm_limb(5);
-
-    env.bitmask_be(&input_limb0, 15, 0, c0);
-    env.bitmask_be(&input_limb0, 30, 15, c1);
-    env.bitmask_be(&input_limb0, 45, 30, c2);
-    env.bitmask_be(&input_limb0, 60, 45, c3);
-    env.bitmask_be(&input_limb0, 75, 60, c4);
-    {
+    let c0_var = {
+        let c0 = Env::get_column_for_msm_limb(0);
+        env.bitmask_be(&input_limb0, 15, 0, c0)
```

Comment:
I was checking inside the code to understand if we're sure it was an open interval (meaning, the 15th is not included, but it goes from 0 to 14). But this was not clarified in the doc. I would suggest to add a comment clarifying that this is the case.

Reply:
Added a3e039fa7cb91dd31a16427a2b570bc0bb0c612a. Feel free to close this thread if it is fine for you.